src/gluonts/model/deepar/_network.py

```diff
@@ -96,9 +99,11 @@ def __init__(
                     else cell
                 )
                 self.rnn.add(cell)
+            self.rnn.cast(dtype=dtype)
```

Comment:
Couldn’t (shouldn’t?) the single blocks in the `HybridSequentialRNNCell` be set to the right dtype?

Reply:
I tried that as well, i.e., casting each cell individually to the right type before adding to the `rnn`. Even that does not force it to use the right dtype for the begin state. I ended up with this version for no particular reason.